pep-0586.rst

```diff
@@ -22,9 +22,15 @@ only expressions that have literally the value "4"::
     def accepts_only_four(x: Literal[4]) -> None:
         pass
 
-    accepts_only_four(4)   # Ok
+    accepts_only_four(4)   # OK
     accepts_only_four(19)  # Rejected
 
+    four = 4
+    accepts_only_four(four)  # Rejected (not a literal)
```

Comment:
Potential wording is "Can be rejected" or "May be rejected" to emphasize that type checkers are allowed to reject this (but are not obliged to reject).

Reply:
But they would be required to reject it if there was intervening code that could possibly assign a different value to `four`. I don't think we need to split such hairs in this initial example.